tests/verifyAzureMySQLPostgreSQLRemoteInstance_test.js

```diff
@@ -0,0 +1,118 @@
+const assert = require('assert');
+
+Feature('Monitoring Azure MySQL and PostgreSQL DB');
+
+Before(async ({ I, settingsAPI, pmmSettingsPage }) => {
+  I.Authorize();
+  await settingsAPI.restoreSettingsDefaults();
+  I.amOnPage(pmmSettingsPage.url);
+});
+
+Scenario(
+  'PMM-T746 - Verify adding monitoring for Azure MySQL, PMM-T744 @instances',
```

Comment:
Please move the test case id at the start, so it becomes easy to read in the reports.